Sources/Diff.swift

```diff
@@ -21,8 +21,8 @@ private struct BoxedHashable<T: NSObjectProtocol>: Hashable {
         self.value = value
     }
 
-    public var hashValue: Int {
-        return self.value.hash
+    public func hash(into hasher: inout Hasher) {
+        hasher.combine(value.hash)
```

Comment:
I think this just needs to be `hasher.combine(value)` but not entirely positive. WDYT?

Reply:
`T` (`value`) itself isn't `Hashable` and things like `MKOverlay` can return a hash integer required by `NSObjectProtocol`, but don't conform to `Hashable` (specifically for Swift objects), so extending the requirements of `T` wouldn't work. I think since these objects are required to return a `hash` this still works nicely. `BoxedHashable` seems to be a wrapper to provide a Swifty version of `NSObjectProtocol`.

Reply:
@freak4pc did you get a chance to review the above? :)

Reply:
Sorry, next time better to tag me if you reply ... otherwise I don't see the notification :)